viite-UI/src/view/ProjectLinkLayer.js

```diff
@@ -212,6 +212,65 @@
     //This will control the double click zoom when there is no selection that activates
     map.on('dblclick', zoomDoubleClickListener);
 
+    var infoContainer = document.getElementById('popup');
+    var infoContent = document.getElementById('popup-content');
+
+    var overlay = new ol.Overlay(({
+      element: infoContainer
+      //autoPan: false,
```

Comment:
yea, we tested that with Jani and forgot it there. if enabled popup would in cases where it is near corners of the map  push map in direction it appears.